src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -44,85 +49,149 @@ public static void reportNetworkBalance(Reporter reporter, double activeGenerati
+        reporter.report(Report.builder()
+                .withKey("busForcedToBePv")
+                .withDefaultMessage("${numBusesWithUpdatedQLimits} bus(es) PQ blocked at their min/max reactive power limit have had their min/max limit updated")
+                .withValue("numBusesWithUpdatedQLimits", String.format("%6s", numBusesWithUpdatedQLimits))
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
-    public static void reportReactiveControllerBusesToPqBuses(Reporter reporter, int switchCount) {
+    public static void reportReactiveControllerBusesToPqBuses(Reporter reporter, int remoteReactivePowerControllerBusToPqCount) {
         reporter.report(Report.builder()
                 .withKey("remoteReactiveControllerBusToPq")
-                .withDefaultMessage("${count} remote reactive power controller buses have switched PQ")
-                .withValue("count", switchCount)
+                .withDefaultMessage("${remoteReactivePowerControllerBusToPqCount} bus(es) with remote reactive power controller have switched PQ")
+                .withValue("remoteReactivePowerControllerBusToPqCount", remoteReactivePowerControllerBusToPqCount)
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
     public static void reportStandByAutomatonActivation(Reporter reporter, String busId, double newTargetV) {
         reporter.report(Report.builder()
                 .withKey("standByAutomatonActivation")
-                .withDefaultMessage("Activation of voltage control of static var compensator with stand by automaton: bus ${busId} switched PQ -> PV with targetV ${newTargetV}")
+                .withDefaultMessage("Activation of voltage control of SVC with stand by automaton: bus ${busId} switched PQ -> PV with targetV ${newTargetV}")
                 .withValue("busId", busId)
                 .withValue("newTargetV", newTargetV)
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
     }
 
+    public static void reportCurrentLimitersPstsChangedTaps(Reporter reporter, int numOfCurrentLimiterPstsThatChangedTap) {
+        reporter.report(Report.builder()
+                .withKey("currentLimiterPstsChangedTaps")
+                .withDefaultMessage("${numOfCurrentLimiterPstsThatChangedTap} current limiters PSTs changed taps")
+                .withValue("numOfCurrentLimiterPstsThatChangedTap", String.format("%6s", numOfCurrentLimiterPstsThatChangedTap))
+                .withSeverity(TypedValue.INFO_SEVERITY)
+                .build());
+    }
+
+    public static void reportActivePowerControlPstsChangedTaps(Reporter reporter, int numOfActivePowerControlPstsThatChangedTap) {
+        reporter.report(Report.builder()
+                .withKey("activePowerControlPstsChangedTaps")
+                .withDefaultMessage("${numOfActivePowerControlPstsThatChangedTap} active power control PSTs changed taps")
```

Comment:
To be consistent with what you did with generator, it should be typed "PST(s)" right?